src/components/index.tsx

```diff
@@ -111,3 +111,22 @@ body.classList.remove('loading');
 // Finally, render the app!
 const appContainer = document.getElementById('app');
 render(<App {...{ store, theme, history }} />, appContainer);
+
+// Register a window resize event listener.
+const oneSecond: number = 1000;
+let resizeRecorderTimeout: number = 0;
+window.onresize = () => {
```

Comment:
Using `onresize` actually overrides other `onresize` handlers.

```suggestion
window.addEventListener('resize', () => {
```